build/build-images.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
```

Comment:
could we please add `set -eu -o pipefail` for all new scripts that we write? 
This allows us to catch a lot of bash traps during development itself.